docs/book/content/OGCore_references.md

```diff
@@ -0,0 +1,5 @@
+# References
+
+```{bibliography} ../OGCore_references.bib
+:style: alpha
```

Comment:
@rickecon What do you think about the `plain` style?  It seems to avoid warnings when a paper has the same authors and year (which results in a duplicate alpha-numeric label).

Reply:
@jdebacker. LaTeX doesn't choke as long as you give the paper a unique reference ID. I wish the Jupyter Book processing did that as well. But I am OK with this citation approach. However, I think the best option is to have the last names of the authors spelled out in the reference, and I don't think this is an option in Jupyter Book.